src/LTOCli/CLI.py

```diff
@@ -84,11 +91,27 @@ def main():
     parser_setNode.add_argument('--network', type=str, nargs=1, required=False, help ='Optional network parameter (ex. L, T)')
     # --------------------------------------------------------------
     parser_sponsorship = subparsers.add_parser('sponsorship', help="Create an Sponsorship Transaction, type 'lto sponsorship --help' for more information")
-    parser_sponsorship.add_argument('option', choices=['create', 'cancel'], help='create / cancel', type=str, nargs=1)
-    parser_sponsorship.add_argument('--recipient', type=str, nargs=1, required = True)
-    parser_sponsorship.add_argument('--account', type=str , nargs=1, required=False, help="Use this option to select one of the account previously stored, The account can be referenced by name or address, if this option is omitted, the default account is used")
-    parser_sponsorship.add_argument('--no-broadcast', action='store_true', required=False, help="Use this option to not broadcast the transaction to the node")
-    parser_sponsorship.add_argument('--unsigned', action='store_true', required=False, help="Use this option to not sign the transaction. Use in combination with the '--no-broadcast' option")
+    sponsorship_subparser = parser_sponsorship.add_subparsers(dest='subparser-name-sponsorship')
+
+
+    parser_sponsorship_create = sponsorship_subparser.add_parser('create',  help='To create a sponsorhip the --recipient is required')
```

Comment:
I see on previous help messages you are using "Create ...", you can remove the "To" here

Reply:
Those are different kind of helps, the previous help="Create were meant to give a description, this help="To create is instead explaining how to use the option, thanks anyway for telling me

Reply:
Yes, my idea was the consistency of how you structure sentences with the same purpose